connectors/access_control.py

```diff
@@ -5,6 +5,26 @@
 #
 
 ACCESS_CONTROL = "_allow_access_control"
+DLS_QUERY = """{
+                    "bool": {
+                        "should": [
+                            {
+                                "bool": {
+                                    "must_not": {
+                                        "exists": {
+                                            "field": "_allow_access_control"
+                                        }
+                                    }
+                                }
+                            },
+                            {
+                                "terms": {
+                                    "_allow_access_control.enum": {{#toJson}}access_control{{/toJson}}
```

Comment:
TIL about the [toJson search template](https://www.elastic.co/guide/en/elasticsearch/reference/current/search-template.html#search-template-convert-json). Nice implementation!